src/client/Microsoft.Identity.Client/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -222,6 +242,34 @@ private SecStatusCode Save(IiOSKey key, string payload)
             return secStatusCode;
         }
 
+        internal SecStatusCode SaveBrokerApplicationToken(string clientIdAsKey, string applicationToken)
```

Comment:
>SaveBrokerApplicationToken [](start = 31, length = 26)

TrySaveBrokerApplicationToken ... and make sure it doesn't throw an exception since you arr returning a status code?